.github/ISSUE_TEMPLATE/documentation-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Documentation Request
+about: Create a request for documentation
+title: ''
+labels: docs
+assignees: ''
+
+---
+
+### Describe what needs to be documented.
+A clear and concise description of what you would like to see.
+
+### Is the documentation missing?
+If so, please describe where have you looked for the documentation and where you expect to see it.
+
+### Is the documentation there but needs improvement?
+
+   *Please provide the url of the documentation.*
```

Comment:
I don't see any reason to indent this line. Seems like it would be consistent with the rest of the templates to not indent it.

Reply:
Done.